lib/HTFeed/JobMetricsCLI.pm

```diff
@@ -0,0 +1,178 @@
+package HTFeed::JobMetricsCLI;
```

Comment:
This whole class was broken out of `HTFeed::JobMetrics`.

Reply:
I think that helps separate responsibilities well.